frontend/app/views/notes/_template.html.erb

```diff
@@ -36,31 +36,12 @@
   </div>
 <% end %>
 
-
-
-<div id="template_note_summary"><!--
-  <div class="subrecord-summary-title">
-    {if type != ""}
-      ${type}
-    {/if}
-    {if label != "" && label != type}
-      {if type != ""} - {/if}
-      ${label}
-    {/if}
-    {if label == "" && type == ""}
-      ${jsonmodel_type}
-    {/if}
-  </div>
-  {if summary.length > 0}
-    <div class="subrecord-summary-content">
-      ${summary}
-    </div>
-  {/if}
---></div>
-
-<% form_note_type ||= nil %>
+<%
+  form_note_type ||= nil
+  nested_in_jsonmodel ||= nil
+%>
   
-<% if ( inline? && form_note_type )  %>
+<% if (inline? && form_note_type) || nested_in_jsonmodel %>
   <% define_template "#{form_note_type}_note_type_selector_inline" do |form, item| %>
```

Comment:
This is the magic selector template that needs to be defined for every "path" -- the reason being that the note types for each context are often different.  So we now use `nested_in_jsonmodel` to output this template for any nested records.